eisbuk-admin/.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+    "eslint.workingDirectories": [
+        {
+            "directory": "./client",
```

Comment:
This should be in gitignore (as it's personal setup and not everyone is using vscode). Maybe add this to the readme file.